src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--days--29">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```
Pay attention to the description
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/a9a4ae3d-dc4f-4add-bf58-5471b7dc19f8)